spec/api/bundes-api.spec.js

```diff
@@ -240,4 +278,60 @@ describe('Bundles API', () => {
                 .expect(400);
         });
     });
+
+
+    describe('put', () => {
+
+        it('should allow to update bundle by id', () => {
+            apiRequest
+                .put('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                .send({
+                    published: 1
+                })
+                .expect(200)
+                .then(() => {
+                    return apiRequest
+                        .get('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                        .expect(res => {
+                            expect(res.body).to.have.property('published')
+                                .and.equal(1);
+                        });
+                });
+        });
+
+        it('should reject update without any data', () => {
+
+            return apiRequest
+                .put('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                .send({})
+                .expect(500);
+        });
+
+        it('should not update id or created_at fields', () => {
+            apiRequest
+                .put('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                .send({
+                    published: 1,
+                    created_at: 100,
+                    id: "foo"
+                })
+                .expect(200)
+                .then(() => {
+                    return apiRequest
+                        .get('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                        .expect(res => {
+                            var body = res.body
+
+                            expect(body).to.have.property('published')
+                                .and.equal(1);
+                            expect(body).to.have.property('id')
+                                .and.equal('79dbdd38-1233-42f8-9d32-a1dad5871b52');
+                            expect(body).to.have.property('created_at')
+                                .and.equal(0);
```

Comment:
it would be more readable with `not.equal(100)`, because that way we dont need to look at fixtures to see reasoning